lib/identity/client.js

```diff
@@ -72,6 +73,26 @@ export default class Client {
     }
   }
 
+  async changePassword(username, oldPassword, newPassword) {
```

Comment:
oldPassword is no longer needed in this signature. I would like to see a helper method exposed for verifying the old password, though :)